### ISSUE TITLE
pyinstaller not packaging image assests in binary

### ISSUE THREAD

**Issue by sclayton33** [PistilliLab/CLAMSwrangler#8]
OS: Manjaro linux
Python version: 3.11

Error:
~~~
Traceback (most recent call last):
  File "main.py", line 235, in <module>
  File "tkinter/__init__.py", line 4129, in __init__
  File "tkinter/__init__.py", line 4074, in __init__
_tkinter.TclError: couldn't open "/path/to/CLAMSwrangler/assets/logo.png": no such file or directory
[18125] Failed to execute script 'main' due to unhandled exception!
~~~

---------------------------

Build program was pyinstaller set to --onefile mode using the command below to generate the .spec file.

~~~
pyi-makespec --onefile main.py
~~~

Then a recursion limit was set in the .spec file, adding two lines to the top.
~~~
# -*- mode: python ; coding: utf-8 -*-
import sys
sys.setrecursionlimit(5000)
~~~

Executable was generated with the following command:
~~~
pyinstaller main.spec
~~~

**Comment by sclayton33** [comment]
Build notes for future reference.

# Linux
Command to create spec file:
~~~
pyi-makespec --onefile --noconsole --add-data "logo.png:." --add-data "icon.ico:." --add-data "icon.png:." --add-data "instructions.txt:." main.py
~~~

Open .spec file and change the NAME to "CLAMS Wrangler v1.0.0" or something like that.

Then run pyinstaller with the .spec file.

~~~
pyinstaller main.spec
~~~

# Windows
Command to create spec file:
~~~
pyi-makespec.exe --onefile --noconsole --add-data "logo.png;." --add-data "icon.png;." --add-data "icon.ico;." --add-data "instructions.txt;." main.py
~~~

Open .spec file and change the NAME to "CLAMS Wrangler v1.0.0" or something like that.

Then run pyinstaller with the .spec file.

~~~
pyinstaller.exe .\main.spec
~~~

**Comment by sclayton33** [comment]
This is fixed as of v1.0.0 on all platforms.
